test/import-style.js

```diff
@@ -626,6 +641,51 @@ test({
 				},
 			}],
 		},
+		// `node:util` only allow `named`, add `default` should keep `named` allowed ... (see next test)
+		{
+			code: `
+				import * as util from "node:util";
+			`,
+			options: [
+				{
+					styles: {
+						'node:util': {
+							default: true,
+						},
+					},
+				},
+			],
+			errors: [{
+				messageId: 'importStyle',
+				data: {
+					allowedStyles: 'named or default',
+					moduleName: 'node:util',
+				},
+			}],
+		},
+		// ...(see previous test), unless we disable `named` explicitly
```

Comment:
Maybe this should not work this way, but this is how it works before, I'm not going to change it.